conf/distro/include/retro-raspberrypi.inc

```diff
@@ -1,16 +1,17 @@
+MACHINEOVERRIDES =. "${@bb.utils.contains('MACHINE_FEATURES', 'full-kms', 'full-kms:', '', d)}"
+
 RPI_ALWAYS_FULLHD ?= "1"
 RPI_ENABLE_AUDIO ?= "1"
+RPI_V4L2M2M_DECODER_rpi_full-kms ?= "1"
```

Comment:
Using `?=` operator with conditional assignment is undefined from what I know (only `=` operator is allowed). 
I think that `rpi` override can be drop for this variable in favor of simple `?=` assignment.

like: 
`RPI_V4L2M2M_DECODER ?= "${@bb.utils.contains('OVERRIDES', 'full-kms', '1', '0', d)}`

Reply:
> Using ?= operator with conditional assignment is undefined from what I know (only = operator is allowed).

It happens to work correctly, I tested something like that 

recipe:
```bash
FOO_rpi ??= "1"
FOO_rpi ?= "2" 
FOO_rpi = "3"
# order test
FOO_rpi ?= "4"
FOO_rpi ??= "5"
```
output:
```
~/.../master/retro-build-environment/build ⑂master* $ bitbake -e conditional-assignment-experiment | grep ^FOO
FOO="3"
FOO_rpi="3"
```

Reply:
After couple of thoughts, beside testing assignment condition.

`RPI_V4L2M2M_DECODER ?=` should be preferred:
* `RPI_V4L2M2M_DECODER = "0"` in `local.conf` will be overridden by assignment condition even this condition is using `?=` or `??=`
* It's hard to change default value with current approach user need to track our overrides and put something like that `RPI_V4L2M2M_DECODER_rpi_full-kms = "0"` in `local.conf`
* `RPI_V4L2M2M_DECODER_rpi_full-kms = "0"` also creates a problem that we need to be backward compatible with exact overrides.